src/index.html

```diff
@@ -5,8 +5,410 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <span>
+        <a href="#">
+          <img
+            class="logo"
+            src="images/logo.png"
+            alt="Moyo"
+          >
+        </a>
+      </span>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Смартфоны и телефоны</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+            Ноутбуки и компьютеры
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Гаджеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Планшеты</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Фото</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">Видео</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="main__cards-container">
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="monoblock-apple"
+          >
+
+          <p
+            class="card__tittle"
+          >
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </p>
+        
+          <p class="card__product-code">Код товара: 195434</p>
+  
+          <div class="card__review-container">
+  
+            <div class="stars card__stars">
```

Comment:
```suggestion
            <div class="card__stars stars stars--4">
```
There should modifier also